redis-lettuce/src/main/java/io/micronaut/configuration/lettuce/RedisConnectionUtil.java

```diff
@@ -31,11 +32,31 @@
 /**
  * Internal utility methods for configuration.
  *
- * @author Graeme Rocher
+ * @author Graeme Rocher, Illia Kovalov
  * @since 1.0
  */
 @Internal
 public class RedisConnectionUtil {
+    /**
+     * Utility method for establishing a redis connection.
+     *
+     * @param beanLocator  The bean locator to use
+     * @param serverName   The server name to use
+     * @param errorMessage The error message to use if the connection can't be found
+     * @return The connection
+     * @throws ConfigurationException If the connection cannot be found
+     */
+    public static AbstractRedisClient findClient(BeanLocator beanLocator, Optional<String> serverName, String errorMessage) {
```

Comment:
don't use Optional as method parameter

Reply:
This is following the pattern of the rest of these util methods...  Would you like them all changed to take a string and perform null checks?